docs: add form to estimate realtime throughput (#20260)
supabase/supabase: apps/docs/pages/guides/realtime/postgres-changes.mdx

There can be a database bottleneck which limits message throughput. If your database cannot authorize the changes rapidly enough, the changes will be delayed until you receive a timeout.

> Please note that currently to maintain the order of changes, we are using a single thread to process changes on the database side. This means that compute upgrades will not improve the performance of this feature by much, refer to form below to estimate the maximum throughput you can expect from your database instance.

Reviewer: ```suggestion
Database changes are processed on a single thread to maintain the change order. That means compute upgrades don't have a large effect on the performance of Postgres change subscriptions. You can estimate the expected maximum throughput for your database below.
```

If you are using Postgres Changes at scale, you should consider using separate "public" table without RLS and filters. Alternatively, you can use Realtime server-side only and then re-stream the changes to your clients using a Realtime Broadcast.

> You can estimate what maximum throughput you can expect from your database instance using the following form:

Reviewer: ```suggestion
Enter your database settings to estimate the maximum throughput for your instance:
```
